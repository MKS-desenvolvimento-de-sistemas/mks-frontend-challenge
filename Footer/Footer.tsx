import styled from "styled-components";

const Container = styled.footer`
  position: fixed;
  bottom: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 34px;
  font-weight: 400;
  font-size: 12px;
  line-height: 15px;
  background-color: #eeeeee;
  color: #000000;
`;

const Footer = () => (
  <Container>MKS Sistemas Â© Todos os direitos reservados</Container>
);

export default Footer;
