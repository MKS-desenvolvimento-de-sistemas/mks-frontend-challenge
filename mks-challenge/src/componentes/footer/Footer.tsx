import React from 'react'

import { 
  FooterStyle

} from './FooterStyled'

function Footer() {
  return (
    <FooterStyle>MKS sistemas Â© Todos os direitos reservados</FooterStyle>
  )
}

export default Footer