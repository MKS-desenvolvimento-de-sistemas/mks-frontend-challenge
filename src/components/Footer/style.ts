import styled from "styled-components";

export const FooterContainer = styled.footer`
  width: 100%;
  height: fit-content;
  text-align: center;
  background-color: var(--grey-0);

  p {
    padding: 15px 0;
    font-weight: 400;
    font-size: 12px;
  }
`;
