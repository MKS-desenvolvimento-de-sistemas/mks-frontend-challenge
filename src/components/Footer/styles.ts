import styled from "styled-components";

export const Footer = styled.footer`
  width: 100%;
  height: var(--footer-height);

  display: flex;
  justify-content: center;
  align-items: center;

  background-color: var(--primary-color-200);

  position: fixed;
  bottom: 0;
  left: 0;
  z-index: var(--z-fixed);

  h5 {
    font-weight: 400;
    font-size: 12px;
    line-height: 15px;
    color: var(--secondary-color-300);
  }
`;
