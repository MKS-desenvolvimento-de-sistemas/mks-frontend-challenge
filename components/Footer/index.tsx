import React from 'react';
import * as C from './styles';

const Footer = () => {
  return (
    <C.FooterWrapper>
      <p>MKS sistemas Â© Todos os direitos reservados</p>
    </C.FooterWrapper>
  )
};

export default Footer;
