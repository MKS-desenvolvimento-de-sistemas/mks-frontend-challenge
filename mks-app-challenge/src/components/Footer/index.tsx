import React from 'react'
import { FooterApp } from './styles';

const Footer = () => {
    return (
        <FooterApp>
            MKS sistemas Â© Todos os direitos reservados
        </FooterApp>
    )
}

export default Footer;