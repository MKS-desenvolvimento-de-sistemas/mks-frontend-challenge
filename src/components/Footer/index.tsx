import * as Styles from "./styles";

const Footer = () => {
  return (
    <Styles.Footer>
      <h5>MKS Sistemas Â© Todos os direitos reservados</h5>
    </Styles.Footer>
  );
};

export default Footer;
