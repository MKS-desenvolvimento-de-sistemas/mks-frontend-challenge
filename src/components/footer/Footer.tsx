import * as S from "./styles";

const Footer = () => {
  return (
    <S.Container>
      <S.FooterText>MKS sistemas Â© Todos os direitos reservados</S.FooterText>
    </S.Container>
  );
};

export default Footer;
