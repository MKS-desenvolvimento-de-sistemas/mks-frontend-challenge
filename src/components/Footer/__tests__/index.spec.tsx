import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Footer from "../";
import renderer from "react-test-renderer";

describe("Footer Component", () => {
  it("should render the footer correctly", () => {
    render(<Footer />);

    const footer = screen.getByTestId("footer");

    expect(footer).toHaveTextContent(
      "MKS Sistemas Â© Todos os direitos reservados"
    );
  });

  it("should match the snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
