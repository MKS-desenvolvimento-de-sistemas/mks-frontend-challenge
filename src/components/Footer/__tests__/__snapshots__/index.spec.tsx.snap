// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match the snapshot 1`] = `
<footer
  className="sc-bczRLJ dntwmJ"
  data-testid="footer"
>
  <h5>
    MKS Sistemas Â© Todos os direitos reservados
  </h5>
</footer>
`;
